src/Diagnostics.DataProviders/AscClient.cs

```diff
@@ -0,0 +1,378 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Diagnostics.DataProviders.Interfaces;
+using Newtonsoft.Json;
+using Diagnostics.ModelsAndUtils.Models.ChangeAnalysis;
+using Diagnostics.DataProviders.TokenService;
+using System.Collections.Generic;
+using System.Net;
+using System.Web;
+using Diagnostics.DataProviders.DataProviderConfigurations;
+
+namespace Diagnostics.DataProviders
+{
+    /// <summary>
+    /// Client to call into Azure Support Center.
+    /// </summary>
+    public class AscClient : IAscClient
+    {
+
+        /// <summary>
+        /// Azure Support Center base uri, initialized from config.
+        /// </summary>
+        private string baseUri;
+
+        /// <summary>
+        /// Azure Support Center api uri, initialized from config.
+        /// </summary>
+        private string apiUri;
+
+        /// <summary>
+        /// Azure Support Center api version, initialized from config.
+        /// </summary>
+        private string apiVersion;
+
+        /// <summary>
+        /// User-agent to pass to Azure Support Center, initialized from config.
+        /// </summary>
+        private string userAgent;
+
+        private string requestId;
+
+        private readonly Lazy<HttpClient> client = new Lazy<HttpClient>(() =>
+        {
+            var client = new HttpClient();
+            client.DefaultRequestHeaders.Accept.Clear();
+            client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            return client;
+        }
+       );
```

Comment:
Looks like the indentation messed up with this on a new line

Reply:
Fixed it :)